### PR TITLE
Fix: Wallet Responsive Web Develop

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.styles.ts [hunk 1]
```diff
@@ -42,7 +45,7 @@ export const WalletBalanceDetailInfoWrapper = styled.div`
     width: 100%;
 
     ${media.mobile} {
-      flex-direction: column;
+      //   flex-direction: column;
```
Comment (jinoosss) [inline packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.styles.ts:48]: Remove unused lines.

File: packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx [hunk 2]
```diff
@@ -50,18 +52,7 @@ async function fetchBalanceDetailInfo(
...
-  };
-
-  useEffect(() => {
-    handleResize();
-    window.addEventListener("resize", handleResize);
-    return () => {
-      window.removeEventListener("resize", handleResize);
-    };
-  }, []);
+  const [breakpoint] = useAtom(CommonState.breakpoint);
```
Comment (jinoosss) [inline packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx:55]: Why didn't you use a variable in `useWindowSize` hook?

**Review by jinoosss — COMMENTED** [review]
LGTM !
I left a few small comments.

File: packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.styles.ts (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/wallet/wallet-balance-detail-info/WalletBalanceDetailInfo.styles.ts:48]: thank you!

File: packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx (hunk 2 above)
Comment (juntaepark) [reply packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx:55]: I checked the design and I don't think I'll need it. Thanks!

Comment (juntaepark) [reply packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx:55]: In the WalletBalanceSummary component, the width is changed at the Mobile breakpoint and is being used.

**Review by jinoosss — APPROVED** [review]
Approve !

**Pull request merged**